### PR TITLE
WEB-750 fix images and ssr

### PR DESCRIPTION
**Pull request by sergivillar**

### CODE REVIEW

File: src/image.tsx [hunk 1]
```diff
@@ -93,7 +98,7 @@ export type ImageProps = {
     dataAttributes?: DataAttributes;
     noBorderRadius?: boolean;
     onError?: (event: React.SyntheticEvent) => void;
-    onLoad?: (event: React.SyntheticEvent) => void;
+    onLoad?: (event?: React.SyntheticEvent) => void;
```
[the remark this answers is not in the document]
Comment (sergivillar) [reply src/image.tsx:101]: mmm true, but, how do we deal with this case in SSR? we do nos have and event there

[the remark this answers is not in the document]
Comment (sergivillar) [reply src/image.tsx:101]: removed, as not used

File: src/__acceptance_tests__/image-ssr-acceptance-test.tsx [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import {openSSRPage} from '../test-utils';
+
+test('ssr list', async () => {
+    const page = await openSSRPage({name: 'image'});
+
+    expect(await page.screenshot()).toMatchImageSnapshot();
```
[the remark this answers is not in the document]
Comment (sergivillar) [reply src/__acceptance_tests__/image-ssr-acceptance-test.tsx:6]: https://jira.tid.es/browse/WEB-775

File: src/image.tsx [hunk 3]
```diff
@@ -143,23 +160,24 @@ const Image = React.forwardRef<HTMLImageElement, ImageProps>(
...
                 }}
+                onLoad={onLoadHandler}
             />
         );
 
+        React.useEffect(() => {
+            // Needed because there is some race condition with SSR and onLoad events
+            // https://github.com/facebook/react/issues/15446
+            if (imageRef.current?.complete) {
+                onLoadHandler();
```
Comment (Marcosld) [inline src/image.tsx:177]: This can be called multiple times if the father doesn't use `React.useCallback`... could that be a problem?

**Review by Marcosld — APPROVED** [review]

**Review by Marcosld — DISMISSED** [review]

File: src/image.tsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (sergivillar) [reply src/image.tsx:177]: I think if that is a problem, the parent component should be the one of wrapping the function with a usecallback, right?

**Review by atabel — APPROVED** [review]

**Pull request merged**